nixos/lib/make-system-tarball.nix

```diff
@@ -22,7 +22,7 @@
   # Extra tar arguments
 , extraArgs ? ""
   # Command used for compression
-, compressCommand ? "pixz"
+, compressCommand ? "pixz -t"
```

Comment:
why aren't we just using `xz -T0`?

Reply:
Interesting, I was not aware of multiprocessor support in xz (`xz -T`), but from cursory testing it would seem that pixz is slightly wall-clock faster (i. e. uses cores more efficiently), especially on high compression settings.

That said, if you suggest it, I can change `pixz` to `xz -T`.

Reply:
I think we can keep `pixz` for now to solve the issue at hand.